### PR TITLE
Fix #5272 - Support router configuration maxMessageSize

### LINKED ISSUE #5272: Support router configuration maxMessageSize
**Overview**

Support router configuration maxMessageSize.  It should be applied to the public configuration.

**Design**

Add this field to the router policy so that is configured on the router.

**Tasklist**

### PR DESCRIPTION
**Pull request by k-wall**
### Type of change

- Enhancement / new feature

### Description

Support router policy maxMessageSize

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Review by rgodfrey — DISMISSED** [review]
LGTM

File: documentation/modules/ref-standard-infra-config-fields.adoc [hunk 1]
```diff
@@ -68,4 +68,5 @@ This table shows the fields available for the standard infrastructure configurat
 |`router.policy.maxSessionsPerConnection` |Specifies the maximum number of sessions allowed per router connection.
 |`router.policy.maxSendersPerConnection` |Specifies the maximum number of senders allowed per router connection.
 |`router.policy.maxReceiversPerConnection` |Specifies the maximum number of receivers allowed per router connection.
+|`router.policy.maxMessageSize` |The maximum size in bytes of AMQP message transfers allowed for this router. This limit is applied to transfers to/from client connections. A value of zero disables this limit.
```
Comment (hmaclean73) [inline documentation/modules/ref-standard-infra-config-fields.adoc:71]:
```suggestion
|`router.policy.maxMessageSize` |Specifies the maximum size in bytes of AMQP message transfers allowed for this router. This limit is applied to transfers to/from client connections. A value of zero disables this limit.
```

**Review by hmaclean73 — CHANGES_REQUESTED** [review]
One minor change.

**Review by hmaclean73 — APPROVED** [review]
LGTM

**Comment by k-wall** [comment]
@enmasse-ci run tests profile=acceptance ocp4

**Pull request merged**